Library/Formula/saldl.rb

```diff
@@ -0,0 +1,39 @@
+class Saldl < Formula
+  desc "CLI downloader optimized for speed and early preview."
+  homepage "https://saldl.github.io"
+  url "https://github.com/saldl/saldl/archive/v31.tar.gz"
+  # version "31"
+  sha256 "146a980ae2109a391c7a8ab6a1c525458db9bf18f3f0477731c9b876630078b4"
+
+  head do
+    # git describe does not work with shallow clones
+    url "https://github.com/saldl/saldl.git", :shallow => false
+  end
+
+  depends_on "pkg-config" => :build
+  depends_on "asciidoc" => :build
+  depends_on "curl"
```

Comment:
You can do `depends_on "curl" if MacOS <= :mavericks` then. Yosemite and El Cap ship a new enough `curl`.